server/events/project_command_builder.go

```diff
@@ -148,6 +149,41 @@ func (p *DefaultProjectCommandBuilder) BuildPlanCommands(ctx *command.Context, c
 	return pcc, err
 }
 
+func (p *DefaultProjectCommandBuilder) BuildPolicyCheckCommands(ctx *command.Context) ([]command.ProjectContext, error) {
```

Comment:
Naming is a bit confusing imo since we use BuildPlanCommands to build Policy check commands as well right?  

Maybe we can make this clearer and call this BuildPolicyCheckReviewComands? 

Note: That doesn't really make sense though either tbh cuz this isn't a command but it's fine for now.

My vote in the first place would have been to just not follow the command runner/builder model here and just write one off code all in the pull_request_review handler itself.  I actually had added the handler stuff later in an attempt to move off all the command runner/builder stuff.

Reply:
We could, but I figured it was cleaner to maintain the legacy model because we would have to inject/rebuild a ton of stuff into the pull request handler like the drainer, locking mechanism, db, etc. This seemed to be the most straightforward in the short-term.